<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WHdlEntry</class>
 <widget class="QWidget" name="WHdlEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>176</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>64</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Обработчик</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="lyTop">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="edPath">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>28</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>Путь</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btRsp">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btMenu">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>..</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="lyBtm">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QTextEdit" name="teAnswer"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="lineBtm">
     <property name="styleSheet">
      <string notr="true">border: 2px solid gray;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="aAsText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Как текст</string>
   </property>
  </action>
  <action name="aAsFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Как файл</string>
   </property>
  </action>
  <action name="aAsQuery">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Как запрос</string>
   </property>
  </action>
  <action name="aSetColor">
   <property name="text">
    <string>Цвет</string>
   </property>
  </action>
  <action name="aQuit">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="aAddHeader">
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="toolTip">
    <string>Добавить заголовок</string>
   </property>
  </action>
  <action name="aDelHeader">
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="toolTip">
    <string>Удалить заголовок</string>
   </property>
  </action>
  <action name="aRequest">
   <property name="text">
    <string>Вызвать</string>
   </property>
   <property name="toolTip">
    <string>Вызвать в браузере</string>
   </property>
  </action>
  <action name="aWaitNot">
   <property name="text">
    <string>Мгновенный</string>
   </property>
  </action>
  <action name="aWaitBtn">
   <property name="text">
    <string>По команде</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>edPath</tabstop>
  <tabstop>teAnswer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
